docs/pages/blog/turbo-1-8-0.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: Turborepo 1.8
+date: 2023/02/22
+description: Turborepo 1.8 brings better flexibility and more control to your codebase by improving turbo's understanding of your workspaces.
+tag: "web development"
+ogImage: /images/blog/turbo-1-8-0/twitter-card.png
+---
+
+import { Tabs, Tab } from '../../components/Tabs'
+
+# Turborepo 1.8
+
+import { Authors } from '../../components/Authors'
+import Badge from '../../components/Badge'
+import Date from "../../components/blog/Date";
+
+<Date>
+  Wednesday, February 22th, 2023
+</Date>
+
+<Authors authors={[
+  'gregsoltis',
+  'nathanhammond',
+  'tomknickman',
+  'anthonyshew',
+  'jaredpalmer',
+  'mehulkar',
+  'chrisolszewski',
+  'nicholasyang',
+  'alexanderlyon'
+]} />
+
+Turborepo 1.8 brings better flexibility and more control to your codebase by improving turbo's understanding of your workspaces.
+
+- [**Workspace Configurations**](#schedule-your-long-running-tasks-with-confidence): You can now add a turbo.json configuration file in a workspace to override the root configuration in your repository.
+- [**Automatic global `turbo` scoping**](#declare-your-outputs-for-improved-clarity): Global turbo now automatically infers your current workspace so that it only runs that workspace’s tasks.
+- [**Easier migrations**](#global-turbo): Automatically migrate to new versions of turbo with `npx @turbo/codemod migrate`.
+
+Update today by running `npx @turbo@codemod migrate`.
+
+## Workspace Configurations
+
+In workspace directories, you can now add a `turbo.json` to:
+
+- add tasks specific to that workspace
+- override configuration for tasks in workspace directories
+
+This will enable teams to scale ownership of their monorepos by moving away from global configuration with fine-grain contol over tasks in workspaces.
```

Comment:
```suggestion
This will enable teams to scale ownership of their monorepos by moving away from global configuration to fine-grain contol over tasks in workspaces.
```